### DISCUSSION TITLE
Is it possible to force Object Caching to Enabled

### DISCUSSION THREAD

**Discussion by curtismchale** [rhubarbgroup/.github#33] [Support (ROC)] [answered]
We're rolling out Redis to a bunch of customer sites on Kubernetes (K8S) and while it's working great, the plugin does not keep object cache enabled when I restart a pod. Pod restarts happen sometimes, and sometimes are manually performed.

I have not found a way to force the Redis caching to be `Enabled` in the settings. I have to head back to the site after each restart and manually enable it again. Is there a `wp-config` constant I'm missing to force it to be enabled? Should I just inspect the database and force the enabled option that way with some other custom code?

We currently do not have  a way to issue `wp cli` commands from our K8S restart setup so I can't use `wp redis enable` to make sure it's enabled after a restart.

**Comment by tillkruss** [comment, marked as answer]
You can `cp redis-cache/includes/object-cache.php wp-content/` and call `redis-cli flushall` afterwards.
